scripts/build_release.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+projectPath=$(dirname `pwd`) 
+folderName=$(basename $(dirname `pwd`)) 
+
+mkdir -p "../../$folderName-cache"
+mkdir -p "../../$folderName-cache/target"
+mkdir -p "../../$folderName-cache/registry"
+
+docker run --env $1 --rm -v "/$projectPath":/code \
+  --mount type=bind,source=/$projectPath-cache/target,target=/target \
+  --mount type=bind,source=/$projectPath-cache/registry,target=/usr/local/cargo/registry \
+  cosmwasm/workspace-optimizer:0.15.1 
```

Comment:
Would be good if we stick to a single scripting type probably using the **Makefile.toml**, what do you think?

Reply:
I am not seeing any Makefile.toml, so this can be refactored to a Makefile.toml by your team afterwards. For me it was only important to have reproductible builds and have the workspace-optimizer version commited to the repository.

For us at ERIS this is our scripting type we are using ;)